### PR TITLE
(Update) Fortify Auth

### PR DESCRIPTION
**Pull request by HDVinnie**

### CODE REVIEW

File: app/Providers/FortifyServiceProvider.php [hunk 1]
```diff
@@ -149,42 +152,56 @@ public function boot(): void
 
             $user = User::query()->where('username', $request->username)->first();
 
+            if ($user && !Hash::check($request->password, $user->password)) {
```
Comment (Roardom) [inline app/Providers/FortifyServiceProvider.php:155]: We shouldn't be hashing the password twice. If we did, we should consider increasing the default bcrypt value. This should be memoized with the following if statement.

File: app/Providers/FortifyServiceProvider.php [hunk 2]
```diff
@@ -149,42 +152,56 @@ public function boot(): void
...
                     $request->session()->invalidate();
 
                     throw ValidationException::withMessages([
-                        Fortify::username() => trans('auth.not-activated'),
+                        Fortify::username() => __('auth.not-activated'),
                     ]);
                 }
 
                 // Check if user is banned
-
```
Comment (Roardom) [inline app/Providers/FortifyServiceProvider.php:165]: Personally, I add an empty line after a comment, when the comment describes code that include empty lines within it, but not going to force anything here.

File: config/auth.php [hunk 3]
```diff
@@ -116,4 +116,18 @@
...
+    |--------------------------------------------------------------------------
+    |
+    | Here you may define the amount of minutes before an email verification
+    | times out and the user is prompted to re-verify their email via the
+    | verify screen. By default, the timeout lasts for one day.
+    |
+    */
+    'verification' => [
+        'expire' => 1440,
+    ],
```
Comment (Roardom) [inline config/auth.php:132]: Where is this used? Couldn't find any documentation about it.

**Review by Roardom — COMMENTED** [review]
Gave it a quick lookover.

File: config/auth.php (hunk 3 above)
Comment (HDVinnie) [reply config/auth.php:132]: https://github.com/laravel/framework/blob/7907bfb50165b3951403e2d9882b4f8e0ee26b9e/src/Illuminate/Auth/Notifications/VerifyEmail.php#L85

Not documented for included in default laravel auth config

**Pull request merged**